Documentation/FrameworkOverview.md

```diff
@@ -8,196 +8,219 @@ learning about new modules and finding more specific documentation.
+
+The `<~` operator can be used to bind properties in different ways. Note that in 
+all cases, the target has to be a `MutablePropertyType`.
+
+* `Property <~ Signal` binds the signal to the property, updating the property's 
+value to the latest value sent by the signal.
+* `Property <~ SignalProducer` creates a signal from the given producer, which will 
+immediately bind to the given property, updating the property's value to the latest 
+value sent by the signal
+* `Property <~ Property` binds the _destination_ property to the latest values 
+the _source_ property.
+
+When bridging to Objective C code (like UIKit or AppKit), `DynamicProperty` can be used to
+wrap a `dynamic` property using Key-Value-Coding and Key-Value-Observing. `DynamicProperty`
+should only be used when KVO/KVC is required by the API used (e.g. `NSOperation`), 
+`MutableProperty` should be preferred whenever possible! 
 
 ## Disposables
 
-The **[RACDisposable][]** class is used for cancellation and resource cleanup.
+A **Disposable**, represented by the [Disposable][] protocol, is a a mechanism
+for memory management and cancellation. 
+
+When starting a [SignalProducer](#signal-producers), a disposable will be returned.
+This disposable can be used by the caller to cancel the work that has been started
+(e.g. background processing, network requests, etc.). Additionally, all temporary 
+resources should be cleaned up and an `Interrupted` event is sent.
 
-Disposables are most commonly used to unsubscribe from a [signal](#signals).
-When a [subscription](#subscription) is disposed, the corresponding subscriber
-will not receive _any_ further events from the signal. Additionally, any work
-associated with the subscription (background processing, network requests, etc.)
-will be cancelled, since the results are no longer needed.
+Observing a [Signal](#signals) may also return a disposable. Disposing it will 
+prevent the observer from receiving any future events from that signal, it will
+however not have any effect on the signal itself.
 
 For more information about cancellation, see the RAC [Design Guidelines][].
 
 ## Schedulers
 
-A **scheduler**, represented by the [RACScheduler][] class, is a serial
-execution queue for [signals](#signals) to perform work or deliver their results upon.
+A **scheduler**, represented by the [SchedulerType][] protocol, is a serial
+execution queue to perform work or deliver results upon.
+
+[Signals](#signals) can be ordered to deliver events a a specific scheduler or
+[SignalProducers](#signal-producers) can be ordered to start their work on 
+a specific scheduler.
 
 Schedulers are similar to Grand Central Dispatch queues, but schedulers support
 cancellation (via [disposables](#disposables)), and always execute serially.
-With the exception of the [+immediateScheduler][RACScheduler], schedulers do not
+With the exception of the [ImmediateScheduler][Scheduler], schedulers do not
```

Comment:
Technically UIScheduler _may_ execute immediately, but I'm not sure it deserves mention here.